libindy/src/services/pool/networker.rs

```diff
@@ -182,7 +183,8 @@ impl PoolConnection {
     fn new(mut nodes: Vec<RemoteNode>, active_timeout: i64, preordered_nodes: Vec<String>) -> Self {
         trace!("PoolConnection::new: from nodes {:?}", nodes);
 
-        thread_rng().shuffle(nodes.as_mut());
+        nodes.shuffle(&mut thread_rng());
+//        thread_rng().shuffle(nodes.as_mut());
```

Comment:
If this is not being used, let's delete this line.

Reply:
+1